test/integration/acceptance/get_roles_test.cpp

```diff
@@ -59,28 +52,26 @@ TEST_F(GetRoles, CanGetRoles) {
  * @then there is should be an exception
  */
 TEST_F(GetRoles, CanNotGetRoles) {
-    auto checkQuery = [](auto &queryResponse) {
-        ASSERT_ANY_THROW(boost::apply_visitor(
-                framework::SpecifiedVisitor<shared_model::interface::RolesResponse>(), queryResponse.get()));
-    };
+  auto checkQuery = [](auto &queryResponse) {
+    ASSERT_ANY_THROW(boost::apply_visitor(
```

Comment:
I think, it's better not to check against throwing just any exception, but use `SpecifiedVisitor` to unwrap response to a specific error type. I assume, it would be StatefulValidationError response. You can check the actual return type in debugger

Reply:
Also, if you implement it, update the comment accordingly